src/app/containers/TopStories/index.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useContext } from 'react';
+import { arrayOf, shape, node } from 'prop-types';
+import SectionLabel from '@bbc/psammead-section-label';
+import styled from 'styled-components';
+import { StoryPromoLi, StoryPromoUl } from '@bbc/psammead-story-promo-list';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+} from '@bbc/gel-foundations/spacings';
+
+import topStories from '#pages/StoryPage/topStories.json';
+import { storyItem } from '#models/propTypes/storyItem';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { GridItemConstrainedLarge } from '#lib/styledGrid';
+import StoryPromo from '../StoryPromo';
+
+const Wrapper = styled(GridItemConstrainedLarge)`
+  margin-bottom: ${GEL_SPACING_DBL};
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    margin-bottom: ${GEL_SPACING_TRPL};
+  }
+`;
+
+const StyledSectionLabel = styled(SectionLabel)`
+  margin-top: 0;
+`;
+
+const TopStories = ({ content }) => {
+  const { script, service, dir } = useContext(ServiceContext);
+  const a11yAttributes = {
+    as: 'section',
+    role: 'region',
+    'aria-labelledby': 'top-stories-heading',
+  };
+  const TopStoriesWrapper = ({ children }) => (
+    <Wrapper {...a11yAttributes}>{children}</Wrapper>
+  );
+  TopStoriesWrapper.propTypes = {
+    children: node.isRequired,
+  };
+  if (!topStories.length) return null;
```

Comment:
This will break if `topStories` is null or undefined. Also would it be better to have this check higher up in the file?

Reply:
`topStories` is fixture data, which comes from the JSON file. It will be removed when the work to consume actual data from the Ares endpoints is done.

Reply:
Do we not still need a check for if the Ares endpoint returns `null`?

Reply:
The Ares endpoint has not been integrated, and depending on how it's integrated the data will come from a prop that is not `topStories`. The check can be done via PropTypes.